cmd/cosign/cli/pivcli/piv_tool.go

```diff
@@ -22,6 +22,7 @@ import (
 	"encoding/json"
 	"flag"
 	"fmt"
+	"os"
```

Comment:
when building with piv-key i was getting

```
CGO_ENABLED=1 go build -tags=pivkey -ldflags "-X github.com/sigstore/cosign/cmd/cosign/cli.GitVersion=cosigned-v0.0.3-dev-35-gd9767fc -X github.com/sigstore/cosign/cmd/cosign/cli.gitCommit=d9767fca7b61b105e61069fa8388363cea28672e -X github.com/sigstore/cosign/cmd/cosign/cli.gitTreeState="clean" -X github.com/sigstore/cosign/cmd/cosign/cli.buildDate='2021-09-15T11:56:00Z'" -o cosign ./cmd/cosign
# github.com/sigstore/cosign/cmd/cosign/cli/pivcli
cmd/cosign/cli/pivcli/piv_tool.go:150:18: undefined: os
^[[A^Cmake: *** [cosign-pivkey] Interrupt: 2
```

Reply:
that's... interesting?

Reply:
There's no os...